healthworkforce/kubernetes/importer/volume/README.md

```diff
@@ -0,0 +1,3 @@
+# Note
+
+These are replicated from healthworkforce/docker/volume as Kubernetes need these to be under the path where it config files are executed from.
```

Comment:
typo `where it config`

Reply:
fixed in https://github.com/openhie/instant/pull/27/commits/2811a62e7f0ab413e1b4b50b9670e17a83400f70